operator/v1/types_console.go

```diff
@@ -143,8 +143,48 @@ type Capability struct {
 	Visibility CapabilityVisibility `json:"visibility"`
 }
 
+// ThemeType defines the type of the theme for the console UI.
+type ThemeType string
+
+// ThemeType values
+const (
+	// DarkTheme represents the dark theme for the console UI.
+	DarkTheme ThemeType = "Dark"
+	// LightTheme represents the light theme for the console UI.
+	LightTheme ThemeType = "Light"
+)
+
+// CustomLogoThemeFile defines a configuration based on theme types for the console UI logo.
+type CustomLogoThemeFile struct {
+	// type specifies the theme type for the console UI. It determines whether the console should use the dark or light theme.
+	// This field is an enum with valid values "Dark" and "Light".
+	// +kubebuilder:validation:Enum:="Dark";"Light"
+	// +required
+	Type ThemeType `json:"type"`
```

Comment:
Maybe `theme` instead of `type`?

Reply:
> Maybe `theme` instead of `type`?
The parent name is `Theme`, so I think it makes sense to leave this as `Type`.

Reply:
I see how this could be a bit confusing having `type` at both `CustomLogo` and `Theme` fields. On the other hand it does make sense, since both are enums.
I would either keep it as is or add full names - `logoType` && `themeType` to avoid any confusion.

Reply:
Using the full names would go from:
```yaml
customLogos:
- type: Masthead
  themes:
  - type: Dark
  ...
```
to:
```yaml
customLogos:
- logoType: Masthead
  themes:
  - themeType: Dark
  ...
```

You would be adding unnecessary repetition because they are nested fields.

Reply:
Right, so you suggest to keep it as it is - `type`, right ?

Reply:
I would